tasks/project-bin.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Add composer_project_path bin directory to global $PATH.
+  template:
+    src: composer-project.sh.j2
+    dest: /etc/profile.d/composer-project.sh
```

Comment:
@christopher-hopper agree that this would be preferable however this would also require adding user context to this role.

However I've it easier to leverage [custom composer commands](https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands).


> Note: Before executing scripts, Composer's bin-dir is temporarily pushed on top of the PATH environment variable so that binaries of dependencies are easily accessible.